tools/c-parser/StrictC.grm

```diff
@@ -1187,6 +1196,8 @@ struct_declaration
                          left basetype, right struct_declarator_list),
                     sdecls)
                  end)
+(* per the C11 spec this should be allowed but let's not for simplicity *)
+(* | static_assert_declaration () *)
```

Comment:
What is the issue here? If we're putting functionality in, but not putting in all of it, would be good to know why.